### DISCUSSION TITLE
[Request] llama对话过程能否保活？

### DISCUSSION THREAD

**Discussion by gaye746560359** [lobehub/lobehub#2260] [Self Hosting] [answered]
### 🥰 需求描述

我是docker部署的ollama，机器是i-3600k+rtx4090
在用lobechat使用llama模型对话时有两处需要优化：
1.启动对话要很久的时间，一直加载中，应该是执行ollama run llama很费时间
2.对话过程暂停一段时间后，再次对话又需要等待很久

### 🧐 解决方案

能否在启动对话后保活模型？或者有其他方案？

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
这可能要问下 ollama 的社区了

[a comment by an automated account is not shown]

**Comment by MapleEve** [comment, marked as answer]
看Ollama 部署的文档，我这边 PR 过对应的环境变量可以让模型保活时间变长

[a comment by an automated account is not shown]
